Mathlib/RingTheory/HahnSeries/Multiplication.lean

```diff
@@ -80,25 +80,26 @@ def HahnModule (Γ R V : Type*) [PartialOrder Γ] [Zero V] [SMul R V] :=
 
 namespace HahnModule
 
-section
-
-variable {Γ R V : Type*} [PartialOrder Γ] [Zero V] [SMul R V]
```

Comment:
What is the reason for inlining these typeclass assumptions?

Reply:
I cannot remember any particular reason.  I have restored the variable declaration inside a small section.